install/kubernetes/cilium/templates/cilium-preflight/clusterrole.yaml

```diff
@@ -40,11 +40,17 @@ rules:
   resources:
   - nodes
   verbs:
-  - update
```

Comment:
Removing update permission here makes the `verbs` empty. Is that intentional? Why not delete the parent as well now?

Reply:
this removal was not intentional.